### PR TITLE
docs: add bullet points about topics on "run in prod".

### PR DESCRIPTION
**Pull request by xavierlepretre**
This PR suggests a list of topics to broach when teaching how to run a Cosmos node in production.

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [x] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [ ] Ready to be merged

### CODE REVIEW

**Comment by CitMC** [comment]
@xavierlepretre should this PR not target the `master` branch?

**Comment by xavierlepretre** [comment]
> @xavierlepretre should this PR not target the `master` branch?

We will need it in the next IDA too so it all depends on the timing for having both branches in sync.

**Comment by CitMC** [comment]
> > @xavierlepretre should this PR not target the `master` branch?
> 
> We will need it in the next IDA too so it all depends on the timing for having both branches in sync.

Ok, will make a note for me to keep an eye on it. 😉

**Comment by xavierlepretre** [comment]
@julienrbrt Your review is keenly awaited.

File: academy/xl-run-prod/5-run.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## As a service
+
+Ideas: https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html#running-via-background-process
+
+## Monitor
+
+About invariants. Ideas: https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html#verify-mainnet
+
+Explorers.
```
Comment (julienrbrt) [inline academy/xl-run-prod/5-run.md:26]: Prometheus and Grafana

**Review by julienrbrt — APPROVED** [review]
lgtm!

**Pull request merged**